WinUIGallery/Samples/ControlPages/Design/IconographyPage.xaml.cs

```diff
@@ -160,4 +176,20 @@ private void TagsItemsView_ItemInvoked(ItemsView sender, ItemsViewItemInvokedEve
             IconsAutoSuggestBox.Text = tag;
         }
     }
+
+    ScrollBarVisibility oldOuterScrollingValue;
+    private void OverrideUpperScrollViewer(bool isEnabled)
+    {
+        var scrollViewer = this.FindAscendant<ScrollViewer>();
+
+        if (isEnabled)
+        {
+            scrollViewer.VerticalScrollBarVisibility = oldOuterScrollingValue;
```

Comment:
![image](https://github.com/user-attachments/assets/67093c29-09e5-4820-8563-c6638eddac77)

This crashes when navigating away from the page

Reply:
Also, would it make sense to turn this in a helperclass? That way we would be able to re-use this code to block page scrolling - that'll be helpful for the colors page where we'd like to have the selectorbar to stick and the content below to scroll.